aruna/api/storage/models/v2/models.proto

```diff
@@ -5,6 +5,14 @@ option go_package = "github.com/ArunaStorage/go-api/v2/aruna/api/storage/models/
 package aruna.api.storage.models.v2;
 
 // --------------- ENUMS ------------------------
+// Defines the public announcement type
+enum AnnouncementType {
+  ANNOUNCEMENT_TYPE_UNSPECIFIED = 0;
+  ANNOUNCEMENT_TYPE_MISC = 1;
```

Comment:
What does misc mean here ? Should we introduce a special "blog" section or similar ?

Reply:
`Misc` was meant for things that are not directly related to the Aruna system, but are more of an organizational nature, such as that the support email address has changed or something. Something more specific, such as `Orga`, would perhaps be better suited for this. Basically, the changes were not meant to be a blog with the public announcements, but that the announcements can be displayed in a blog-like way.

If we think that interesting facts in and around Aruna could also be interesting for the users, the enum can of course also be extended with `Blog`.